CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contributing to Zonemaster::Backend
+
+Contribution to this repository is welcome. Contribution can be either an issue
+report or a code or a documentation update. Also see the information in the
+[main README][Zonemaster/Zonemaster README] in the main Zonemaster respository.
+
+## Issue
+
+First search for a similar issue in the [issues list]. If a relevant issue is
```

Comment:
Maybe suggest looking at the pull requests as well?

Reply:
As above.